frontend/src/metabase/admin/settings/components/SettingsPages/GeneralSettingsPage.unit.spec.tsx

```diff
@@ -69,7 +77,8 @@ const setup = async () => {
     </>,
   );
 
-  await screen.findByText("Redirect to HTTPS");
+  // NOTE: For cloud plans, "Site name" will appear, so we assert for it instead of "Redirect to HTTPS"
+  await screen.findByText(isCloudPlan ? "Site name" : "Redirect to HTTPS");
```

Comment:
not a fan of conditional assertions in tests if they can be avoided. can you find another element to assert on that isn't variable?

Reply:
Addressed all feedback via https://github.com/metabase/metabase/pull/66469/commits/0dc1ba941d0b04265b43c40135edf85caa9b0cce 👍🏻